sdk-core/src/main/java/care/data4life/sdk/record/RecordContract.kt

```diff
@@ -30,8 +30,7 @@ import io.reactivex.Completable
 import io.reactivex.Single
 import org.threeten.bp.LocalDate
 
-interface RecordContract {
-
+class RecordContract {
```

Comment:
Why was this changed to a class?

Reply:
Good catch..this one seemed to slipped while rebasing.

Reply:
done